client/src/app/+about/about.component.html

```diff
@@ -1,3 +1,12 @@
-<p>
-  about works!
-</p>
+
+    <h3> אודות</h3>
+    <p>נכון לסוף 2013, כטריליון שקלים מנוהלים בקרנות הפנסיה וקופות הגמל בישראל.</p>כספים אלו - המיועדים להבטחת רמת חיינו לאחר גיל הפרישה לפנסיה - מנוהלים ע"י הגופים המוסדיים בישראל
```

Comment:
@eyalcohen4 Make sure the text is bold like in: https://drive.google.com/drive/u/0/folders/0B7RkqHKoREPqRnZiRnZoWFF1ek0